libs/pilight/core/mail.c

```diff
@@ -359,6 +359,11 @@ int sendmail(char *host, char *login, char *pass, unsigned short port, struct ma
 			error = -1;
 			goto close;
 		}
+		if(strncmp(recvBuff, "500", 1) == 0) {
+			logprintf(LOG_NOTICE, "SMTP: neg. response - abort");
```

Comment:
A 5xx reply code is fatal, for some replys (most notable 551), parsing of the reponse is required, but not implemented, so i decided to simply terminate the request.